GPU/Common/TextureCacheCommon.cpp

```diff
@@ -1197,7 +1197,6 @@ bool TextureCacheCommon::GetCurrentFramebufferTextureDebug(GPUDebugBuffer &buffe
 }
 
 void TextureCacheCommon::NotifyConfigChanged() {
-	clearCacheNextFrame_ = true;
```

Comment:
What if all you do is remove this line? This actually didn't seem to happen before the same way, though I don't see why it would be crashy.

Reply:
Tested removing and that fixes the crash. Just turned everything off, I can try to move it tomorrow.